components/nexus-repository/src/main/java/org/sonatype/nexus/repository/cache/CacheController.java

```diff
@@ -44,7 +46,15 @@ public CacheController(final int contentMaxAgeMinutes, @Nullable final String ca
    * carrying same token as created in this method.
    */
   public void invalidateCache() {
-    this.cacheToken = newCacheToken();
+    invalidateCache(newCacheToken());
+  }
+
+  /**
+   * After invoking this method, all {@link #isStale(CacheInfo)} checks will return true that has {@link CacheInfo} not
+   * carrying same token as created in this method.
+   */
+  public void invalidateCache(final String cacheToken) {
+    this.cacheToken = checkNotNull(cacheToken);;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)